ionoscloud/resource_server_test.go

```diff
@@ -225,6 +225,20 @@ func TestAccServerBasic(t *testing.T) {
 					resource.TestCheckResourceAttrPair(constant.DataSource+"."+constant.ServerResource+"."+constant.ServerDataSourceById, "nics.0.ipv6_ips.2", constant.ServerResource+"."+constant.ServerTestResource, "nic.0.ipv6_ips.2"),
 				),
 			},
+			{
+				Config: testAccCheckServerConfigShutdown,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(constant.ServerResource+"."+constant.ServerTestResource, "vm_state", "SHUTOFF"),
```

Comment:
search and replace all the places where you have hardcoded, with constants

Reply:
hardcoded values replaced